beets/ui/commands.py

```diff
@@ -1479,17 +1479,20 @@ def move_items(lib, dest, query, copy, album, pretend, confirm=False):
             show_path_changes([(obj.path, obj.destination(basedir=dest))
                                for obj in objs])
     else:
-        if confirm:
-            objs = ui.input_select_objects(
-                u'Really %s' % act, objs,
-                lambda o: show_path_changes(
-                    [(o.path, o.destination(basedir=dest))]))
+        if export:
+            util.copy(item.path, item.destination(basedir=dest))
```

Comment:
This error:
https://travis-ci.org/beetbox/beets/jobs/222222532#L894

says that the tests are calling the `move_items` function without providing the new argument you've added. That gives you two options: you can either make the flag optional (provide a default value), or you can fix the tests to provide it every time they call the function. It would also be nice to have a test for this new functionality!

It might be helpful for you to set up the testing stuff locally so you can catch bugs during your development. There's more detail here:
https://github.com/beetbox/beets/wiki/Testing

But the short version is that you should install and run `nosetests`.